docs/source/tracking/tracking-api.rst

```diff
@@ -0,0 +1,281 @@
+| ``mlflow.project.env``        | The runtime context used by the MLflow project.                                        |
+|                               | Possible values: ``"docker"`` and ``"conda"``.                                         |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.project.entryPoint`` | Name of the project entry point associated with the current run, if any.               |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.docker.image.name``  | Name of the Docker image used to execute this run.                                     |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.docker.image.id``    | ID of the Docker image used to execute this run.                                       |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.log-model.history``  | Model metadata collected by log-model calls. Includes the serialized                   |
+|                               | form of the MLModel model files logged to a run, although the exact format and         |
+|                               | information captured is subject to change.                                             |
++-------------------------------+----------------------------------------------------------------------------------------+
+
+Custom Tags
+~~~~~~~~~~~
+
+The :py:func:`MlflowClient.set_tag() <mlflow.client.MlflowClient.set_tag>` function lets you add custom tags to runs. A tag can only have a single unique value mapped to it at a time. For example:
+
+.. code-block:: python
+
+  client.set_tag(run.info.run_id, "tag_key", "tag_value")
+
+
+Get MLflow Run instance from autologged results
+-----------------------------------------------
+
+In some cases, you may want to access the MLflow Run instance associated with the autologged results, similarly to you can get `mlflow.start_run()` 
+You can access the most recent autolog run through the :py:func:`mlflow.last_active_run` function. Here's a short sklearn autolog example that makes use of this function:
+
+.. code-block:: python
+
+    import mlflow
+
+    from sklearn.model_selection import train_test_split
+    from sklearn.datasets import load_diabetes
+    from sklearn.ensemble import RandomForestRegressor
+
+    mlflow.autolog()
+
+    db = load_diabetes()
+    X_train, X_test, y_train, y_test = train_test_split(db.data, db.target)
+
+    # Create and train models.
+    rf = RandomForestRegressor(n_estimators=100, max_depth=6, max_features=3)
+    rf.fit(X_train, y_train)
+
+    # Use the model to make predictions on the test dataset.
+    predictions = rf.predict(X_test)
+    autolog_run = mlflow.last_active_run()
```

Comment:
Can we show an example of what the return type is and show the stdout return of `mlflow.last_active_run()`?